episodes/11-exercises.md

```diff
@@ -114,33 +142,96 @@ and that assesses their ability to do so.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-Well-designed exercises are one of the most valuable resources for an Instructor:
-time spent on this is well invested.
+## Demo of Writing an Exercise 
+
+Well-designed exercises are one of the most valuable resources for an instructor and 
+any time spent on this is well invested.
+
+<!--- will there be an example lesson that the trainer is working on ? -->
+
+To create an exercise in the Carpentries lesson infrastructure, 
```

Comment:
Optional: I'm trying to make "Carpentries Workbench" a thing, but "lesson infrastructure" is perfectly fine.

```suggestion
To create an exercise in the Carpentries Workbench, 
```